src/plugins/home/public/application/components/feature_directory.js

```diff
@@ -89,6 +89,7 @@ export class FeatureDirectory extends React.Component {
   renderTabs = () => {
     return this.tabs.map((tab, index) => (
       <EuiTab
+        data-test-subj={tab.name.replace('&', '').replace(/\s/g, '')}
```

Comment:
I'd suggest to use the tab's id here, then you would not need a regexp, and it's less code so like 
{`tab-${tab.id}`}
what do you think?

Reply:
Done thank you!